### ISSUE TITLE
Typographic styles

### ISSUE THREAD

**Issue by taisdesouzalessa** [mozilla/network#59]
Send sheet in sketch file with typographic styles + mockups with applied styles.

**Comment by taisdesouzalessa** [comment]
<img width="345" alt="screen shot 2017-02-03 at 9 58 05 am" src="https://cloud.githubusercontent.com/assets/22157921/22602298/817c4282-e9f7-11e6-823b-a7ac8d730410.png">

@gvn the sheet in in Google Drive (just shared the folder with you): "all-wireframes-typography-dev"

Please notice that when you select the style in Sketch you can see its name, which is the same as noted in the typographic style sheet, so you can use the sketch file as a reference without going back and forth to the style page. 

<img width="727" alt="screen shot 2017-02-03 at 10 04 56 am" src="https://cloud.githubusercontent.com/assets/22157921/22602522/6d17ce96-e9f8-11e6-8c7d-d20bf72e1340.png">

cc: @sabrinang @xmatthewx

**Comment by taisdesouzalessa** [comment]
@gvn mockups are not ready yet (we are working on assets, alignment and spacings), just consider typographic styles for now and structure, please :).

**Comment by xmatthewx** [comment]
Thanks @taisdesouzalessa @sabrinang. 

Let's please discuss the use of arrows on the buttons and links. What's the rationale for this? To me, the arrows indicate you're going to send me to a different website, or I'm going to enter a new space. So, the button from Projects to Pulse seems like an okay use. But going to another page on the site or submitting a form seems strange. Do you feel strongly that we should be using them?

**Comment by gvn** [comment]
@taisdesouzalessa – Please review https://mozilla.github.io/network/style-guide/ and let me know if anything needs to be revised.

**Comment by taisdesouzalessa** [comment]
@gvn I made some tweaks in the sheet above based on the last screens we made: "Menu" and some tweaks in "People" page. Now we have 2 new styles and 2 tweaks in the sheet. The new file is in the server: _PROJECTS/Mozilla Network Site/dev/all-wireframes-ui - The styles sheet has the label "NEW" in the new elements and a little explanation on the ones I changed.

I am focusing on delivering the mobile screens for now but I will review the typefaces in the staging as soon as I have some time, together with @sabrinang
